pkg/pillar/types/volumetypes.go

```diff
@@ -58,17 +61,14 @@ func (config VolumeConfig) LogModify(logBase *base.LogObject, old interface{}) {
 	}
 	if oldConfig.ContentID != config.ContentID ||
 		oldConfig.MaxVolSize != config.MaxVolSize ||
-		oldConfig.RefCount != config.RefCount ||
-		oldConfig.GenerationCounter != config.GenerationCounter {
+		oldConfig.RefCount != config.RefCount {
 
 		logObject.CloneAndAddField("content-id", config.ContentID).
 			AddField("max-vol-size-int64", config.MaxVolSize).
 			AddField("refcount-int64", config.RefCount).
-			AddField("generation-counter-int64", config.GenerationCounter).
 			AddField("old-content-id", oldConfig.ContentID).
 			AddField("old-max-vol-size-int64", oldConfig.MaxVolSize).
 			AddField("old-refcount-int64", oldConfig.RefCount).
-			AddField("old-generation-counter-int64", oldConfig.GenerationCounter).
```

Comment:
Don't you want to log changes to the local and normal generation counter?

Reply:
The counters are part of the key so any change in them will trigger Delete/Create, not Modify.